worker/src/RTC/TransportCongestionControlClient.cpp

```diff
@@ -106,31 +122,45 @@ namespace RTC
 	{
 		MS_TRACE();
 
-		this->rtpTransportControllerSend->packet_sender()->InsertPacket(packetInfo.length);
-		this->rtpTransportControllerSend->OnAddPacket(packetInfo);
+		if (this->rtpTransportControllerSend != nullptr)
+		{
+			this->rtpTransportControllerSend->packet_sender()->InsertPacket(packetInfo.length);
+			this->rtpTransportControllerSend->OnAddPacket(packetInfo);
+		}
```

Comment:
Cosmetic, 

Let's leave if the conditions are not met for readability purposes.

```suggestion
		if (this->rtpTransportControllerSend == nullptr)
		{
		        return;
		}
		
		this->rtpTransportControllerSend->packet_sender()->InsertPacket(packetInfo.length);
		this->rtpTransportControllerSend->OnAddPacket(packetInfo);		
```